### PR TITLE
Ddan 106 bugfix updating status

### PR DESCRIPTION
**Pull request by Armstrong-SuHwan**
### PURPOSE OF PR
* Simple summary of this PR
- fix: fix bug when finished task, status remained "Running" 

### What type of PR is it?
* fix

### JIRA ISSUE
* https://ddangcoonfire.atlassian.net/jira/software/c/projects/DDAN/boards/1?selectedIssue=DDAN-106
* https://ddangcoonfire.atlassian.net/jira/software/c/projects/DDAN/boards/1?selectedIssue=DDAN-109

### HAVE YOU CHECKED ...
* [X] ADD JIRA ID ON PR TITLE (ex. [DDAN-1234] PR TITLE)
* [X] pre-commit
* [X] Branch Format ( type/issue-NUM ) (ex. refactoring/DDAN-1234 )

### Explanation
( Adding screenshot would be great help )

### CODE REVIEW

File: experiment_scheduler/master/process_monitor.py [hunk 1]
```diff
@@ -73,15 +73,19 @@ def _health_check(self, thread_queue, time_interval=1):
...
                         if self.selected_task_manager == -1:
                             self.selected_task_manager = 1
-                    if len(server_status.task_id_array) > 0:
-                        for task_id in server_status.task_id_array:
-                            task = TaskEntity.get(id=task_id)
-                            task.status = 2
-                            task.commit()
+
+                    for task_msg in server_status.task_status_array:
+                        self.logger.warning(task_msg.task_id, task_msg.status)
```
Comment (ptaejoon) [inline experiment_scheduler/master/process_monitor.py:84]: Plase remove unused log.

File: experiment_scheduler/task_manager/task_manager_server.py [hunk 2]
```diff
@@ -129,22 +129,26 @@ def use_gpu(self):
...
 
+    @start_end_logger
     def health_check(self, request, context):
         """
         Return current server status and finished task id list which are finished
         """
+
         server_status = ServerStatus()
         server_status.alive = True
+        server_status.task_status_array.extend([])
```
Comment (ptaejoon) [inline experiment_scheduler/task_manager/task_manager_server.py:140]: Seems like extend does nothing in this line.

**Review by ptaejoon — COMMENTED** [review]
Thanks for your effort fixing the issues and adding more detailed explanation in README.md !

File: experiment_scheduler/task_manager/task_manager_server.py (hunk 2 above)
Comment (Armstrong-SuHwan) [reply experiment_scheduler/task_manager/task_manager_server.py:140]: this line is necessary for appending task status

**Pull request merged**